content/spire/try/oidc-federation-aws.md

```diff
@@ -0,0 +1,415 @@
+               ]
+           }
+       ]
+   }
+   ```
+
+5. Click **Review policy**.
+
+6. For **Name**, type the name `oidc-federation-test-policy`.
+
+7. Click **Create policy**.
+
+## Create an AWS IAM Role for the Identity Provider
+
+The IAM role contains the connection parameters for the OIDC federation to AWS such as the OIDC identity provider, IAM policy, and SPIFFE ID of the connecting workloads.
+
+1. Navigate to the AWS [Identity and Access Management (IAM) page](https://console.aws.amazon.com/iam/home#/home), logging in if necessary.
+
+2. Click **Roles** on the left and then click **Create Role** in the middle of the page.
+
+3. Click **Web Identity** near the top of the page.
+
+4. For **Identity provider**, choose the identity provider that you created in AWS. The identity provider will be your Discovery Provider FQDN followed by `:aud`, such as `oidc-discovery.example.org:aud`.
+
+5. For **Audience**, choose the audience you specified in the identity provider: `mys3`.
+
+6. Click **Next: Permissions**.
+
+7. Search for the policy that you created in the previous section: `oidc-federation-test-policy`. Click the check box next to that policy and then click **Next: Tags**. (Don't click the name of the policy.)
+
+8. Click **Next: Review** to skip the **Add Tags** screen.
+
+9. Type the name `oidc-federation-test-role` for the IAM role and click **Create role**. 
+
+## Add the SPIFFE ID to the IAM Role
+
+To allow the workload from outside AWS to access AWS S3, add the workload's SPIFFE ID to the IAM role. This restricts access to the IAM role to JWT SVIDs with the specified SPIFFE ID.
+
+1. Navigate to the AWS [Identity and Access Management (IAM) page](https://console.aws.amazon.com/iam/home#/home), logging in if necessary.
+
+2. Click **Roles** on the left, use the search field to find the `oidc-federation-test-role` IAM role that you created in the last section, and click the role.
+
+3. Click the **Trust relationships** tab near the middle of the page and then click **Edit trust relationship**.
+
+4. In the JSON access control policy, add a condition line at the end of the `StringEquals` attribute to restrict access to workloads matching the workload SPIFFE ID that was assigned in the [Kubernetes Quickstart](/spire/try/getting-started-k8s/). The new line to add is:
+
+   ```json
+   "MY_DISCOVERY_DOMAIN:sub": "spiffe://example.org/ns/default/sa/default"
+   ```
+   But substitute your Discovery Provider FQDN for MY\_DISCOVERY\_DOMAIN. Add a comma at the end of the previous line after `"mys3"`. When finished, the JSON should look similar to:
+
+   ```json
+   {
+     "Version": "2012-10-17",
+     "Statement": [
+       {
+         "Effect": "Allow",
+         "Principal": {
+           "Federated": "arn:aws:iam::012345678901:oidc-provider/MY_DISCOVERY_DOMAIN"
+         },
+         "Action": "sts:AssumeRoleWithWebIdentity",
+         "Condition": {
+           "StringEquals": {
+             "MY_DISCOVERY_DOMAIN:aud": "mys3",
```

Comment:
This confused me since the text above says this should have been substituted for something. I'd suggest doing the substitution here and noting above something like "for example, if you used the domain `oidc-discovery.example.org` to host your OIDC discovery endpoint"

Reply:
Agreed, done.